### PR TITLE
Added documentation for PackageDescription

### LINKED ISSUE #8578: Package metadata properties does not document correct package description attributes
The documentation for the properties used to generate the NuGet metadata is out of date in regards to the package description. The [link](https://developercommunity.visualstudio.com/content/problem/266675/package-metadata-properties-are-missing-from-sdk-p.html?childToView=364299#comment-364299) discusses the specific issue I'm mentioning here.

The section on NuGet properties says to use the `Description` attribute. However this appears to be a fallback attribute. The `PackageDescription` attribute, if defined, is used to set the package description in the NuSpec file. If it does not exist then `Description` is used. However `Description` is also used for the assembly description, which is often different.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: cb4428fc-215c-78e1-5ff0-fe73f7a08260
* Version Independent ID: 6741b1de-38a4-fbb1-d79d-acc47d549a2d
* Content: [Additions to the csproj format for .NET Core](https://docs.microsoft.com/en-us/dotnet/core/tools/csproj)
* Content Source: [docs/core/tools/csproj.md](https://github.com/dotnet/docs/blob/master/docs/core/tools/csproj.md)
* Product: **dotnet-core**
* GitHub Login: @blackdwarf
* Microsoft Alias: **mairaw**

### PR DESCRIPTION
**Pull request by CoolDadTx**
## Summary
Added documentation for PackageDescription. Adjusted documentation for Description to convey its dual purpose and when it is used. 

Fixes #8578

### CODE REVIEW

File: docs/core/tools/csproj.md [hunk 1]
```diff
@@ -194,9 +194,13 @@ A human-friendly title of the package, typically used in UI displays as on nuget
 ### Authors
 A semicolon-separated list of packages authors, matching the profile names on nuget.org. These are displayed in the NuGet Gallery on nuget.org and are used to cross-reference packages by the same authors.
 
-### Description
+### PackageDescription
+
 A long description of the package for UI display.
 
+### Description
+A long description of the package for UI display. This property is used if `PackageDescription` is not specified. It is also used to specify the generated assembly's description.
```
Comment (Thraka) [inline docs/core/tools/csproj.md]: Thank you for opening this PR! 

After reading through your linked visual studio feedback and the original page, I think we can improve this even further. According to the [AssemblyInfo section](https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#assemblyinfo-properties), the **Description** property is really the description of the assembly I think we should describe it that way first and then state that this property will be used as the package description of that **PackageDescription** property is not specified.

**Review by Thraka — CHANGES_REQUESTED** [review]

File: docs/core/tools/csproj.md (hunk 1 above)
Comment (CoolDadTx) [reply docs/core/tools/csproj.md]: I agree.

Comment (CoolDadTx) [reply docs/core/tools/csproj.md]: Sorry for the delay. I updated the description. 

I noticed that the attributes defined on this page are very inconsistent in how they describe things. Examples:
Authors - A semicolon-separated list
Copyright - Copyright details
PackageOutputPath - Determines the
IncludeSymbols - This Boolean value
Istool  - Specifies whether

I used the terminology that PackageDescription was using but I think this page could stand to have the descriptions reviewed to make them read like they are not copied from documentation elsewhere.

File: docs/core/tools/csproj.md [hunk 2]
```diff
@@ -194,9 +194,13 @@ A human-friendly title of the package, typically used in UI displays as on nuget
 ### Authors
 A semicolon-separated list of packages authors, matching the profile names on nuget.org. These are displayed in the NuGet Gallery on nuget.org and are used to cross-reference packages by the same authors.
 
-### Description
+### PackageDescription
+
 A long description of the package for UI display.
 
+### Description
+A long description for an assembly. This property is used if `PackageDescription` is not specified.
```
Comment (Thraka) [inline docs/core/tools/csproj.md]: We're close 😄 

As it stands right now, it just sounds like the `Description` property is ignored if the `PackageDescription` property is set. I think we need it made clear that we're saying this property will **also** be used for the generated package if the `PackageDescription` is not specified.

Also, I agree with you about all of the descriptions.

**Review by Thraka — CHANGES_REQUESTED** [review]

File: docs/core/tools/csproj.md (hunk 2 above)
Comment (CoolDadTx) [reply docs/core/tools/csproj.md]: OK, how does this sound?

**Comment by Thraka** [comment]
@CoolDadTx PERFECT. Thanks! 😃

**Review by Thraka — APPROVED** [review]

**Pull request merged**